docs/integration-guide.md

```
@@ -0,0 +1,159 @@
+  throw new Error("Unlicensed access");
+}
+// Proceed with unlock flow
+```
+
+---
+
+## License Verification Flow (Unlock Service)
+
+The full license verification challenge-response flow:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)